### PR TITLE
Update xochitl.oxide to support templatectl

### PR DESCRIPTION
**Pull request by Eeems**

### CODE REVIEW

**Comment by pr0fsmith** [comment]
Tested.  All is well.

File: package/xochitl/package [hunk 1]
```diff
@@ -5,7 +5,7 @@
 pkgnames=(xochitl)
 pkgdesc="Read documents and take notes"
 url=https://remarkable.com
-pkgver=0.0.0-10
+pkgver=0.0.0-11
 timestamp=2021-11-20T23:31Z
```
Comment (LinusCDE) [inline package/xochitl/package]:
```suggestion
timestamp=2021-11-29T22:06Z
```

**Review by LinusCDE — DISMISSED** [review]
Tested on rM 2 and works fine. Could add a custom template with templatectl and it showed up when launched through oxide.

You just forgot to update the timestamp. I based the time on your first commit that changed the version.

**Comment by Eeems** [comment]
@LinusCDE updated

**Review by raisjn — APPROVED** [review]
based on two people saying it works

**Pull request merged**